faqs/isitselfpromotion.markdown

```diff
@@ -0,0 +1,20 @@
+---
+layout: page
+title: Is it self promotion?
+permalink: /faqs/isitselfpromotion
+---
+
+Our Code of Conduct discourages self-promotion, but definining self-promotion is somewhat of a grey area.
+
+> Don't spam the channels with links, or links to pages, for the purpose of monetization of user views and which provide no value to the community
+
+If you think your own project provides value and interest to the community, you can sometimes share it anyway, even if you have a financial incentive for doing so.
+
+Some pointers:
+
+* If you're new to the community, people are going to be a lot less welcoming to even mild self-promotion. If they know who you are, they might be more welcoming of it. Think about why you are on ZATech. If you're primarily here because you heard it's a good place to reach South African tech professionals, you're probably spamming. If you're around to ask and answer questions, discuss things, etc and happen to do some self promotion on the side, it might be OK.
+* Read the room: different channels will be accepting of different levels of self-promotion. For example, `#developers` is generally pretty anti any kind of self-promotion while `#shameless-advertising` is open to most kinds of self-promotion other than clear spam/scam messages. Channels like `#startups` are somewhere in between. Read the previous messages and get to know some of the more familiar faces before posting self-promotional stuff.
```

Comment:
I don't think being open to "scam messages" should be ok :D

Reply:
yeah that's what it says? 😝

> most kinds of self-promotion **other than** clear spam/scam messages.